build.gradle

```diff
@@ -705,8 +705,8 @@ project('standalone') {
         main = "io.pravega.local.LocalPravegaEmulator"
         classpath = sourceSets.main.runtimeClasspath
         systemProperties System.getProperties()
-        systemProperties 'logback.configurationFile' : new File('config/logback.xml').absolutePath
-        systemProperties 'singlenode.configurationFile' : new File("config/standalone-config.properties").absolutePath
+        systemProperties 'logback.configurationFile' : new File("${project.rootDir}/config/logback.xml")
```

Comment:
The PR description mentions:

```
absolutePath does not return the same result for JDK 8 and JDK 11.
```

is there any gradle documentation describing the change of behavior?

Reply:
The issue is https://github.com/gradle/gradle/issues/5187